docs/guide/seo.rst

```diff
@@ -0,0 +1,57 @@
+================================
+Search Engine Optimization (SEO)
+================================
+
+What is search engine optimization?
+-----------------------------------
+
+Search engines drive a significant portion of site traffic for docs that publish as public-facing websites.
+Search engine optimization (SEO) describes the means and methods used to have your web pages display for a set of desired search terms.
+
+The work of SEO demands familiarity with the ever-changing rules of the algorithms search engines use to select and arrange web sites to return for any particular query.
+These algorithms change frequently, so the requirements for achieving the best optimization also change frequently.
+
+Fortunately, good tech writing already does most of the standard practices required for excellent SEO.
```

Comment:
```suggestion

SEO for documentation
------------------------

Good tech writing follows most of the standard practices required for excellent SEO.
```

(so that we call out the documentation-specific considerations)

Reply:
Great suggestion